code/go/pkg/validator/validator_test.go

```diff
@@ -28,6 +28,7 @@ func TestValidateFile(t *testing.T) {
 	}{
 		"good":                               {},
 		"good_v2":                            {},
+		"good_input":                         {},
```

Comment:
Added a new `good_input` package to add these keys. It is copied from the `sql_input` test package.